README.md

```diff
@@ -13,11 +13,24 @@ Use Kymatio if you need a library that:
 * supports 1-D, 2-D, and 3-D wavelets, and
 * runs seamlessly on CPU and GPU hardware.
 
-Website: [http://www.kymat.io](http://www.kymat.io)
-
 
 ## Installation
 
+### Important links
+
+* Official source code repo: [https://github.com/kymatio/kymatio](https://github.com/kymatio/kymatio)
+* Download releases: https://pypi.org/project/????????
```

Comment:
there is a chicken-and-egg situation here. we to be registered on PyPI to put this on the README, and we need a README to register on PyPI.
I would be in favor of removing this line and adding it on the beta

Reply:
We will add it in timethen!